llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -352,6 +352,9 @@ class TargetTransformInfo {
   unsigned getInliningCostBenefitAnalysisSavingsMultiplier() const;
   unsigned getInliningCostBenefitAnalysisProfitableMultiplier() const;
 
+  /// \returns The bonus of inlining the last call to a static function.
+  int getInliningLastCallToStaticBonus() const;
```

Comment:
I'm not sure if it makes sense to use `unsigned` as return value, but `InlineConstants::LastCallToStaticBonus` is `int`.

Reply:
Also, not sure if it is better to make it as `getInliningLastCallToStaticBonus(const CallBase *CB)`.